ajenti/middleware.py

```diff
@@ -2,6 +2,7 @@
 import time
 import random
 import gevent
+import pyotp
```

Comment:
The PyOTP deb file is not available before ubuntu 16.10. If a package is not available as deb or rpm file, then how is it handled?